pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file:
+
+`composer init`
+
+- Create new project from an existing package:
+
+`composer create-project {{project_name}} {{target_path}} {{version}}`
+
+- Install packages specified in composer.json file:
```

Comment:
Ok, I think anyone with basic level of experience of composer would understand  this, but yes I can enforce it.